app/jobs/sync_publisher_promo_stats_job.rb

```diff
@@ -3,12 +3,17 @@ class SyncPublisherPromoStatsJob < ApplicationJob
   include PromosHelper
   queue_as :transactional
 
-  def perform(promo_id: active_promo_id)
-    # Might be able to select fewer with a more specific query
-    publishers = Publisher.where(promo_enabled_2018q1: true)
+  # Syncs promo stats for all publishers by default
+  # If a publisher is provided, sync stats for only that publisher
 
-    publishers.find_each do |publisher|
+  def perform(promo_id: active_promo_id, publisher: false)
```

Comment:
I think it would be clearer to leave the `false` out as the default value and have `publisher` set to nil by default. The code should run the same way, but with `false` I think my other choice is `true` and not a Publisher.